tests/helpers/deployments/test_deployments.py

```diff
@@ -343,32 +327,6 @@ def test_generates_deploy_oracle_contract(self) -> None:
                     continue
                 app_instance.generate_agent(0)
 
-
-class TestValidates(BaseDeploymentTests):
```

Comment:
Where are these tests now?

Reply:
we've removed the functionality  which was being tested so we don't need these anymore